addons/web/views/report_templates.xml

```diff
@@ -308,7 +312,7 @@
                     </script>
                 </t>
             </head>
-            <body class="container" t-att-onload="subst_needed and 'subst()'">
+            <body class="container" t-att-onload="subst_needed and 'subst()'" style="overflow:hidden">
```

Comment:
Instread of custom style, you could try to use bootstrap classes:
overflow-hidden for example. Does not work the same way? That way, it will be easier to update in the future.
Same for style="margin-left: auto" you can use ml-auto.  
Check other templates if you can do that too.

Reply:
For some reason wkhtmltopdf doesn't like some bootstrap classes, like ml-auto. Changing the style to inline margin-left auto solved the issue.